compiler/rustc_smir/src/stable_mir/ty.rs

```diff
@@ -285,3 +299,10 @@ pub struct TraitDecl {
     pub implement_via_object: bool,
     pub deny_explicit_impl: bool,
 }
+
+pub type ImplTrait = EarlyBinder<TraitRef>;
+
+pub struct TraitRef {
+    pub def_id: ImplDef,
```

Comment:
This should be a `TraitDef`, as the `DefId` refers to the trait, not the impl

Reply:
Uff, right ... my bad.